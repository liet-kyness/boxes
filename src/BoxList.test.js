import BoxList from "./BoxList";
import { render, fireEvent } from "@testing-library/react";

it('renders without crashing', () => {
    render(<BoxList />);
});

it('matches snapshot', () => {
    const { asFragment } = render(<BoxList />);
    expect(asFragment()).toMatchSnapshot();
});