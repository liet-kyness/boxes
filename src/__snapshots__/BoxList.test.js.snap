// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <form>
        <div>
          <label
            for="height"
          >
            height
          </label>
          <input
            id="height"
            name="height"
            type="text"
            value=""
          />
        </div>
        <div>
          <label
            for="width"
          >
            width
          </label>
          <input
            id="width"
            name="width"
            type="text"
            value=""
          />
        </div>
        <div>
          <label
            for="backgroundColor"
          >
            background color
          </label>
          <input
            id="backgroundColor"
            name="backgroundColor"
            type="text"
            value=""
          />
        </div>
        <button
          id="newBoxButton"
        >
          add box
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
